src/pages/_app.js

```diff
@@ -1,9 +1,11 @@
-import WebXLayout from '../layouts/WebXLayout';
-import '../styles/globals.css';
-
+import WebXLayout from "../layouts/WebXLayout";
+import "../styles/globals.css";
+import SEO from "../../next-seo.config";
+import { DefaultSeo } from "next-seo";
 function MyApp({ Component, pageProps }) {
```

Comment:
Let's add a empty line here, nothing but just a style preference

```suggestion
import { DefaultSeo } from "next-seo";

function MyApp({ Component, pageProps }) {
```